firebase-perf/src/main/java/com/google/firebase/perf/metrics/NetworkRequestMetricBuilder.java

```diff
@@ -61,7 +61,7 @@
   private final GaugeManager gaugeManager;
   private final TransportManager transportManager;
 
-  private final NetworkRequestMetric.Builder networkRequestMetricBuilder =
+  private final NetworkRequestMetric.Builder networkMetricBuilder =
```

Comment:
Should we call it builder? According to suggestions in the ticket

Reply:
Done!